restructure the Getting Started nav (#465)
neo4j/docs-aura: modules/ROOT/pages/getting-started/quick-start-guide.adoc

== Create an account

To access Neo4j Aura, you need to have an Aura account.
Navigate to link:https://console.neo4j.io[Neo4j Aura console] and follow the instructions for registration.
Once you have accepted the terms and conditions and verified your email address, you can start using the console.

The console exists in two versions, the classic experience, and the early access new console, as mentioned in xref:index.adoc[About Neo4j Aura console].
You can toggle between the different versions, via the account dropdown in the top right corner. 

== Create an instance 

The first step is to select the plan that best suits your needs.
You can upgrade your plan later if your needs change.

Once you've selected a plan, your instance is created.
A password is generated for the instance, ensure to either copy or download it, as it will **not** be recoverable after.
The password is required to access your instance later.

[NOTE]
====
You can only create **one Free instance** per account.
To create more instances, you need to upgrade your plan.
See link:https://neo4j.com/pricing/[Neo4j Pricing] for more information on the different plans.
====

== Connect to an instance

> :description: This page describes how to connect to an instance using Neo4j AuraDB.

Reviewer: ```suggestion
```

> To work with a database in an instance, you’ll first need to establish a connection.

Reviewer: ```suggestion
To interact with a database in an instance, you need to establish a connection.
```

. Go to *Import*, *Explore* or *Query*. 
. Select *Status* and from there you can connect to an instance.
. You may need your *Username* and *Password* credentials.

[.shadow]
.Connection banner
image::connectionbanner.png[]

[.shadow]
.Connection modal
image::connectionauthentication.png[]

> .Connection modal fields explained

Reviewer: ```suggestion

```
